.github/workflows/extract-translation-source-files.yml

```
@@ -149,6 +149,22 @@ jobs:
           # extract_translations into EN_DIR and adds them
           DJANGO_PATH=$(find $EN_DIR -name 'django.po')
           DJANGOJS_PATH=$(find $EN_DIR -name 'djangojs.po')
+          ############## Special support for XBlocks
+          # if both files are not found, then it can be an XBlock (thus, text.po and textjs.po files)
+          if [ -z "$DJANGO_PATH" ] && [ -z "$DJANGOJS_PATH" ]; then
+            DJANGO_PATH=$(find $EN_DIR -name 'text.po')
+            DJANGOJS_PATH=$(find $EN_DIR -name 'textjs.po')
+            # Rename the files to django.po and djangojs.po, if exists
+            if [ ! -z "$DJANGO_PATH" ]; then
```

```
$ man test
NAME
       test - check file types and compare values
...
       -n STRING
              the length of STRING is nonzero
...
```


Therefore the nitpick: 
```suggestion
            if [ -n "$DJANGO_PATH" ]; then
```